### PR TITLE
LIVE: 1249-Header Image Football Scores component 

### PR DESCRIPTION
**Pull request by dskamiotis**
## Why are you doing this?

I have  opened this PR to get the ball rolling and provide platform for feedback.
I have begun by implementing the design.

Header Image Football Scores component - we need to display the match report scores and teams in the HeaderImage component. (https://theguardian.atlassian.net/browse/LIVE-1249)

- [x] Added a yellow background div to house score and teams
- [x] Used `${brandAltBackground.primary}` and `${body.medium({ fontWeight: 'regular' })};` for consistent styling
- [ ] Use the `maybeRender` function so the <div> and scores only show up on match reports
- [ ] Gets scores and teams from `mobile-sport`

**note** * I have temporarily added dummy scores and teams to test design
## Changes

Added a yellow background div to house score and teams
Used `${brandAltBackground.primary}` and `${body.medium({ fontWeight: 'regular' })};` for consistent styling

## Screenshots

| Target | Before | After |
| --- | --- | --- |
|  |  |  |

### CODE REVIEW

File: src/components/editions/headerImage.tsx [hunk 1]
```diff
@@ -12,6 +13,15 @@ import type { FC } from 'react';
...
 const styles = css`
 	margin: 0;
+	position: relative;
+	div {
+		background-color: ${brandAltBackground.primary};
+		position: absolute;
+		bottom: 0;
+		left: 0;
+		${body.medium({ fontWeight: 'regular' })};
+		padding: 0px 5px 3px 5px;
```
Comment (JamieB-gu) [inline src/components/editions/headerImage.tsx:23]: Prefer `remSpace` over `px`.

Comment (dskamiotis) [reply src/components/editions/headerImage.tsx:23]: 👍

Comment (dskamiotis) [reply src/components/editions/headerImage.tsx:23]: Cant quite get the precise px conversion with remSpace as `remSpace[1]` = 0.25 and `remSpace[1]` = 0.5, I would have to use `rem` to specify 0.3125 for 5px

Comment (JamieB-gu) [reply src/components/editions/headerImage.tsx:23]: I'd suggest just converting to Source values:

```suggestion
		padding: 0px ${remSpace[1]} ${remSpace[1]} ${remSpace[1]};
```

and seeing how that looks.

**Comment by JamieB-gu** [comment]
Closing in favour of #1173.

**Pull request closed without merging**